system/View/Table.php

```diff
@@ -265,7 +265,7 @@ protected function _prepArgs(array $args)
         // If there is no $args[0], skip this and treat as an associative array
         // This can happen if there is only a single key, for example this is passed to table->generate
         // array(array('foo'=>'bar'))
-        if (isset($args[0]) && count($args) === 1 && is_array($args[0]) && ! isset($args[0]['data'])) {
```

Comment:
Why was the `data` excluded?

Reply:
It will cause the `setHeading` function to be executed incorrectly when the heading field contain named `data`.

Reply:
The `&& ! isset($args[0]['data'])` came from this commit: https://github.com/bcit-ci/CodeIgniter/commit/f0b69a85
Are you absolutely sure that just removing it does not cause another issue?
Frankly, I don't get the code yet.

Reply:
I am not sure but the all of the test cases get passed.
And I also don't get where will use this condition and the data processing of the array field contains `data` so far.

Or we can add one more process to check that whether the heading array has been handled correctly when the table is being generated.

Reply:
> the all of the test cases get passed.

Yes. But we cannot see the coverage in Coveralls.

> coverage/coveralls — Target branch is ahead of PR base. Rebase the PR to fix. 

Can you rebase the PR branch?

Reply:
> Can you rebase the PR branch?

Done.

Reply:
Based on my usage of the table class (quite extensive) I have not noticed any issues thus far with the patch applied.

Reply:
Thank you all!